File: Services/Dto/DestinyCollectDeliverDto.cs
using System;
using System.Collections.Generic;

namespace Services.Dto
{
    public class DestinyCollectDeliverDto
    {
        public int Id { get; set; }
        public int DestinyClientId { get; set; }
        public ClientDto DestinyClient { get; set; }
        public int DestinyPartnerId { get; set; }
        public PartnerDto DestinyPartner { get; set; }
        public string NoRegisterName { get; set; }
        public string NoRegisterAddress { get; set; }
        public List<CollectDeliverDto> CollectsDelivers { get; set;}
    }
}

File: Views/src/app/_components/administrative/local/out-sourced/dto/destiny-collect-deliver-dto.ts
import { ClientDto } from "../../../client/dto/client-dto";
import { CollectDeliverDto } from "./collect-deliver-dto";
import { PartnerDto } from "./partner-dto";

export class DestinyCollectDeliverDto {
  id: number;
  destinyClientId: number;
  destinyClient: ClientDto;
  destinyPartnerId: number;
  destinyPartner: PartnerDto;
  noRegisterName: string;
  noRegisterAddress: string;
  collectsDelivers: CollectDeliverDto[];
}
